text/fuzz/corpus/from_test_parse_error_0

```diff
@@ -0,0 +1 @@
+bla 3.14
```

Comment:
Please end any text files with a single newline character (http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline)

(or does that go against the purpose of the corpus items?)